extensions/trenit/src/departures-and-arrivals.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Action,
+  ActionPanel,
+  Color,
+  Icon,
+  LaunchProps,
+  List,
+  LocalStorage,
+  showToast,
+  Toast,
+  useNavigation,
+} from "@raycast/api";
+import { createDeeplink } from "@raycast/utils";
+import { getStations } from "./api/stations-service";
+import { StationView } from "./station-view";
+import { Station } from "./models/station";
+
+import { useState, useEffect } from "react";
+
+const stations = getStations();
+
+function StationItem({
+  station,
+  toggleFavorite,
+  isFavorite = false,
+}: {
+  station: Station;
+  toggleFavorite: (id: string) => void;
+  isFavorite?: boolean;
+}) {
+  return (
+    <List.Item
+      title={station.name}
+      icon={isFavorite ? { source: Icon.Star, tintColor: Color.Yellow } : undefined}
+      actions={
+        <ActionPanel>
+          <Action.Push title="Show Trains" target={<StationView station={station} />} icon={Icon.Train} />
+          <Action
+            title={isFavorite ? "Remove from Favorites" : "Add to Favorites"}
+            shortcut={{ modifiers: ["ctrl"], key: "a" }}
```

Comment:
```suggestion
            shortcut={Keyboard.Shortcut.Common.Pin}
```
Could we use this hotkey?

Reply:
oh yeah, missed this docs page. Added this, and even the one for refresh in the detail view. Thanks for pointing it out :D